### ISSUE TITLE
Partial parameters

### ISSUE THREAD

**Issue by d316** [SmartBear/ruby-handlebars#17]
Hi. Thanks for this awesome gem! Are there any plans to add support for partial parameters (https://handlebarsjs.com/partials.html#partial-parameters)?

**Comment by d316** [comment]
Hi. I tried adding the feature but to no avail. Could you take a look at this [branch](https://github.com/d316/ruby-handlebars/tree/partial_parameters) and see what may be the issue? Thanks!

**Comment by d316** [comment]
Parameter(s) weren't designed to parse partial arguments. Cf. pull request #19 that adds this feature.

**Comment by vincent-psarga** [comment]
Your PR is merged and should be available in the next release :)

(Just copying the note I added in the PR - it's closed so it might not be read there)

One thing I'm wondering about the argument: do you think it would be interesting to allow for safe helper use ?

Something like that:
```ruby
      it 'with helper as argument' do
        hbs.register_helper('wrap_parens') {|context, value| "(#{value})"}
        hbs.register_partial('with_args', "[{{fname}} {{lname}}]")
        expect(evaluate("Hello {{> with_args fname='jon' lname=(wrap_parens 'doe')}}")).to eq("Hello [jon (doe)]")
      end
```

**Comment by d316** [comment]
Hi Vincent. I added support for this and variables in the new pull request. Please consider merging. Dan

**Comment by vincent-psarga** [comment]
That looks great indeed. I see you just made changes to get the tests working. I merge that once it works and get the 0.4 on its way ;)

**Comment by d316** [comment]
It seems that the Github Travis app is hanging. All tests pass locally and also shows as passing on Travis [https://travis-ci.org/SmartBear/ruby-handlebars/builds/600629658](https://travis-ci.org/SmartBear/ruby-handlebars/builds/600629658).

**Comment by vincent-psarga** [comment]
That's now available in 0.4.0 :) thanks again for your contribution, that's really cool features :)
